package.json

```diff
@@ -25,6 +25,7 @@
     "tslint": "^5.10.0",
     "tslint-consistent-codestyle": "^1.13.1",
     "tslint-eslint-rules": "^5.3.1",
-    "tslint-microsoft-contrib": "^5.0.3"
+    "tslint-microsoft-contrib": "^5.0.3",
+    "@fimbul/bifrost": "^0.9.0"
```

Comment:
In what sense is this a dependency? Where is it being used and for what?

Reply:
Npm asked me to install it at one point. It seems to be a dependency of `tslint-consistent-codestyle` so I guess it's no longer needed...